kiota_authentication_azure/azure_identity_access_token_provider.py

```diff
@@ -33,6 +33,7 @@ def __init__(
         options: Optional[Dict],
         scopes: List[str] = [],
         allowed_hosts: List[str] = [],
+        is_cae_enabled: Optional[bool] = True,
```

Comment:
doesn't the default value make it optional for the caller already?

Reply:
Yes, I think we should not make it Optionally and use default value of true to enable it by default without user having to explicitly set